### ISSUE TITLE
./configure: line 5859: syntax error near unexpected token `disable-static'

### ISSUE THREAD

**Issue by laapsaap** [grke/burp#408]
I am compiling the latest release and I am getting this on CentOS6 and MacOS. I removed it and it compiles perfectly. Just not sure if it has impact or not.

```
./configure: line 5859: syntax error near unexpected token `disable-static'
./configure: line 5859: `LT_INIT(disable-static)'
```

**Comment by grke** [comment]
Hello,
I think it is probably something to do with your libtool/automake/autoconf packages.
I don't think that just removing the line is a problem.

Maybe try the CentOS/MacOS equivalent of:
apt-get install autoconf automake libtool
apt-get install make pkg-config check g++ librsync-dev libz-dev libssl-dev uthash-dev libyajl-dev
autoreconf -vif
Then configure.

**Comment by laapsaap** [comment]
running latest versions autoconf 2.69 and automake 1.15    above error still happens.

**Comment by grke** [comment]
Hello,
I ran it on my mac VM a week ago and it was fine.
I also have mac builds triggering via travis with every commit and they are fine too.
Maybe have a look in .travis.yml in the burp source to see what it does.

brew uninstall libtool
brew install libtool
?

**Comment by laapsaap** [comment]
Hi grke, 

Thank you for your quick reply as usual.

I am not using brew as I am compiling autoconf/automake/libtool and all other stuff myself in /usr/local.   /usr/local is my default path.

I tried everything, well its not a big big problem because I just remove LT_INIT(disable-static) from the configure file after I run autoreconf.  But at first I thought the problem was that I had some old version of something related to this. But I updated everything I think is used for this.  Oh well!
